src/api/app/models/token/workflow.rb

```diff
@@ -53,13 +56,17 @@ def validation_errors
       error_messages = []
 
       error_messages << @scm_webhook.errors.full_messages unless @scm_webhook.valid?
-      @workflows.each do |workflow|
+      @workflows.try(:each) do |workflow|
         error_messages << workflow.errors.full_messages unless workflow.valid?
       end
 
       error_messages.flatten
     end
   end
+
+  def ping?
```

Comment:
This should be a method in the [SCMWebhook](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/scm_webhook.rb) class. It doesn't belong here.